src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -328,6 +328,108 @@ It is recommended to use this when reporting any issue.
 
 `subctl gather --module broker,service-discovery --type resources`
 
+### `cloud`
+
+#### `cloud prepare`
+
+`subctl cloud prepare [flags]`
+
```

Comment:
I think that it will be more clear and consistent with the rest of this page if you leave this section of
#### `cloud prepare` 
empty (skip line `subctl cloud prepare [flags]` since it's an emtpy cmd)